frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -169,47 +286,47 @@ export default {
           break
       }
 
-      if (!this.proposal.votes) {
-        return this.proposal.ownProposal
+      if (this.votes.received < 2) {
+        return this.isOwnProposal
```

Comment:
Why "2" ? "1" I understand, now "2" ? ... 🤔

Reply:
the proposal gets 1 'for' vote when proposed (the proposer's vote), so 2 here means the proposer + 1 :) maybe I can subtract the proposer's vote from the received votes and go back to 1 here, that would be more self-explanatory?